File: Python Automation Code/health.py
import unittest
from selenium import webdriver
import time

class Health(unittest.TestCase):
    def setUp(self):
        self.driver = webdriver.Chrome()

    def test_submission(self):
        driver = self.driver

        driver.get('https://hidden-chamber-33381.herokuapp.com/#/')
        driver.maximize_window()                                                                      #Maximize the window
        assert 'health' in driver.title
        time.sleep(5)

        signin = driver.find_element_by_xpath(' //a[@class = \'alert-link\'] ')                       #Sign in Link
        signin.click()
        time.sleep(3)

        user_name  = driver.find_element_by_id('username')                                            #Username
        user_name.send_keys("admin")
        time.sleep(2)

        password = driver.find_element_by_id('password')                                              #Password
        password.send_keys("admin")                            
        time.sleep(2)

        submit = driver.find_element_by_xpath(' //button[@class= \'btn btn-primary\'] ')              #Submit
        submit.click()
        time.sleep(10)


    def teardown(self):
        self.driver.close()

if __name__ == '__main__':
    unittest.main()

File: Java Code/Health.java
package siddhi;

import org.openqa.selenium.By;
import org.junit.Assert;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.chrome.ChromeDriver;
import org.openqa.selenium.support.ui.Wait;
import org.openqa.selenium.support.ui.WebDriverWait;


public class Health {
	static WebDriver driver;
	static Wait<WebDriver> wait;

	public static void main(String args[]) {
		
		driver = new ChromeDriver();
		//wait = new WebDriverWait(driver, 5);
		
		driver.get("https://hidden-chamber-33381.herokuapp.com/#/");
		Assert.assertEquals("health", driver.getTitle());
		
		driver.manage().window().maximize();
		wait = new WebDriverWait(driver, 3);
		
		driver.findElement(By.xpath("//a[@class = \'alert-link\']")).click();
		wait = new WebDriverWait(driver, 5);
		
		driver.findElement(By.id("username")).sendKeys("admin");
		wait = new WebDriverWait(driver, 3);
		
		driver.findElement(By.id("password")).sendKeys("admin");
		wait = new WebDriverWait(driver, 3);
		
		driver.findElement(By.xpath("//button[@class= \'btn btn-primary\']")).click();
		wait = new WebDriverWait(driver, 3);
		System.out.println("Login Successfull");
		
		wait = new WebDriverWait(driver, 5);
		//driver.close();
	}
	   
}